Formula/icu4c.rb

```diff
@@ -40,6 +40,11 @@ def install
   end
 
   test do
-    system "#{bin}/gendict", "--uchars", "/usr/share/dict/words", "dict"
+    if File.readable? "/usr/share/dict/words"
```

Comment:
Your explanation makes it seem like this would be better
```suggestion
    if File.exists? "/usr/share/dict/words"
```

Reply:
Makes sense, I've changed it.